extras/parse_travis_log.py

```diff
@@ -177,6 +180,35 @@ def process_static_analysis(f, line):
         if line.startswith(' - clang-format for '):
             d.update(process_clang_format(f, filename))
 
+def process_pep8(f, line):
+    """
+    Process PEP8 output for a certain file.
+    """
+    # Check PEP8 on file pynest/nest/lib/hl_api_connections.py:
+    filename = line.split(' ')[-1].strip()[0:-1]
+    d = set()
+    res = {filename: d}
+    while True:
```

Comment:
In fact, no -- the underlying file handles moves forward, so:
````
f = open(file, 'r')
f1 = iter(f)
f2 = iter(f)
for l in f1: print l
````

will move f2 forward as well as f1. But in general for iterators, you would be right. And you're probably right that everything should just move the iterator instead of the file handle around... but it is too wide for this pr.

I'd just do:
````
for line in f: ....
````

since that's anyway the same thing.